src/runtime/virtcontainers/qemu_amd64.go

```diff
@@ -173,6 +186,10 @@ func (q *qemuAmd64) cpuModel() string {
 		cpuModel += ",vmx=off"
 	}
 
+	if q.sgxEPCSize != 0 && !q.vmFactory {
+		cpuModel += ",+sgx"
```

Comment:
is this necessary with `defaultCPUModel`?

Reply:
you are right, thanks :+1: